README.md

```diff
@@ -1,3 +1,5 @@
+Changes to `desktop-ui.json` affects only Lantern desktop clients. The corresponding sections in `messages.json` are ignored by new desktop clients. All Android related configurations are in `messages.json`.
```

Comment:
I kept forgetting which file to modify, and here's what I found by checking the code.